theme/imagecaption/theme.scss

```diff
@@ -2,27 +2,14 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 @import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_colors.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_shadow.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_states.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_spacing.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_fonts.scss';
 
 .ck-widget.image {
```

Comment:
Is `.ck-editor` applied in inline editor? It'd be best if the class was as short as possible, but I think it can't be `.ck-editor`.

If we would like to use something shorter than `.ck-editor__editable`, then perhaps we can start using an additional class `.ck-content` for the editables?